BookingsApi/BookingsApi.Contract/Requests/Enums/HearingRole.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace BookingsApi.Contract.Requests.Enums
```

Comment:
could we move this in a separate folder instead of the enums folder? Looks like we only have enums in here.

Reply:
done